### DISCUSSION TITLE
What will happen to my accounts if simplelogin shuts down?

### DISCUSSION THREAD

**Discussion by oniedzialek** [simple-login/app#1643] [Q&A] [answered]
Hi! 
I am wondering if I will lose control of my internet accounts if SimpleLogin shuts down in the future. This is a theoretical question, but very important for me. 
Thank you for your answers!

**Comment by t2noob** [comment, marked as answer]
Are you talking about the simple login domains? you would probably lose control of those accounts since you would not be able to gain control of the domain. If you have your own domain you could easily transfer them to another emailing system or even a self hosted version of simplelogin and transfer your domain emails with a simple *catch all on the domain. One of the reasons it's great to have a private domain on the simplelogin platform if you ever feel like jumping ship or something happens to the service you could easily go to other email systems and have very minimal downtime on your emails. (Basically, as fast as you can change DNS records and wait for them to  get propagated)

**Reply by oniedzialek** [reply]
That make sens. Awesome option. Thank you!

**Reply by ghost** [reply]
Not really. Custom domains may be banned on many websites. Here is the [proof](https://github.com/simple-login/app/discussions/1550)

Website should allow to register with a well known domain but restrict you from registering if you’re using custom domain
